### PR TITLE
ClusterPodStatuses analyzer

### PR DESCRIPTION
**Pull request by sgalsaleh**

### CODE REVIEW

File: pkg/analyze/cluster_pod_statuses.go [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+				r.URI = outcome.Pass.URI
+				results = append(results, &r)
+				whens = append(whens, outcome.Pass.When)
+			}
+		}
+
+		fmt.Println("whens", whens)
+
+		if len(results) == 0 {
+			return nil, errors.New("empty outcomes")
```
Comment (divolgin) [inline pkg/analyze/cluster_pod_statuses.go]: this should be not an error, right?

File: pkg/analyze/cluster_pod_statuses.go [hunk 2]
```diff
@@ -0,0 +1,141 @@
...
+			}
+			pods = append(pods, nsPods...)
+		}
+	}
+
+	results := []*AnalyzeResult{}
+
+	for _, pod := range pods {
+		whens := []string{}
+		results := []*AnalyzeResult{}
```
Comment (syntastical) [inline pkg/analyze/cluster_pod_statuses.go]: Don't think you meant to shadow the variable in the outer scope

**Review by divolgin — APPROVED** [review]

File: schemas/collector-troubleshoot-v1beta2.json [hunk 3]
```diff
@@ -432,33 +432,6 @@
                   }
                 }
               },
-              "kernelModules": {
```
Comment (syntastical) [inline schemas/collector-troubleshoot-v1beta2.json:435]: I take it that this is just some older unrelated stuff that needed to be cleaned up
